sktime/classification/deep_learning/resnet.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+"""Multi Layer Perceptron Network (MLP) for classification."""
+
+__author__ = ["James-Large", "AurumnPegasus", "nilesh05apr"]
+__all__ = ["ResNetClassifier"]
+
+from sklearn.utils import check_random_state
+
+from sktime.classification.deep_learning.base import BaseDeepClassifier
+from sktime.networks.resnet import ResNetNetwork
+from sktime.utils.validation._dependencies import _check_dl_dependencies
+
+_check_dl_dependencies(severity="warning")
+
+
+class ResNetClassifier(BaseDeepClassifier):
+    """
+    Residual Neural Network as described in [1].
+
+    Parameters
+    ----------
+    n_epochs       : int, default = 1500
+        the number of epochs to train the model
+    batch_size      : int, default = 16
+        the number of samples per gradient update.
+    random_state    : int or None, default=None
+        Seed for random number generation.
+    verbose         : boolean, default = False
+        whether to output extra information
+    loss            : string, default="mean_squared_error"
+        fit parameter for the keras model
+    optimizer       : keras.optimizer, default=keras.optimizers.Adam(),
+    metrics         : list of strings, default=["accuracy"],
+    activation      : string or a tf callable, default="sigmoid"
+        Activation function used in the output linear layer.
+        List of available activation functions:
+        https://keras.io/api/layers/activations/
+    use_bias        : boolean, default = True
+        whether the layer uses a bias vector.
+    optimizer       : keras.optimizers object, default = Adam(lr=0.01)
+        specify the optimizer and the learning rate to be used.
+
+
+    Notes
+    -----
+    Adapted from the implementation from source code
+    https://github.com/hfawaz/dl-4-tsc/blob/master/classifiers/resnet.py
+
+    References
+    ----------
+        .. [1] Wang et. al, Time series classification from
+    scratch with deep neural networks: A strong baseline,
+    International joint conference on neural networks (IJCNN), 2017.
+
+    Examples
+    --------
+    >>> from sktime.classification.deep_learning.resnet import ResNetClassifier
+    >>> from sktime.datasets import load_unit_test
+    >>> X_train, y_train = load_unit_test(split="train")
+    >>> clf = ResNetClassifier(n_epochs=20, bacth_size=4) # docktest: +SKIP
+    >>> clf.fit(X_train, Y_train) # docktest: +SKIP
```

Comment:
`# doctest: +SKIP` to have them skipped, another small typo is causing doctest failure in the checks.